apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1208,6 +1208,31 @@ static void fs_vfs_ioctl_tc(void)
 }
 
 /**
+* @testcase         libc_stdio_dprintf_tc
+* @brief            Exact analogs of fprintf and vfprintf, except that they output to a file descriptor fd instead of to a stdio stream.
+* @scenario         Exact analogs of fprintf and vfprintf, except that they output to a file descriptor fd instead of to a stdio stream.
+* @apicovered       dprintf
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_dprintf_tc(void)
+{
+	char *filename = VFS_FILE_PATH;
+	char *str = VFS_TEST_CONTENTS_1;
+	int fd;
+	int ret;
+
+	fd = open(filename, O_RDWR);
+	TC_ASSERT_GEQ("open", fd, OK);
+
+	ret = dprintf(fd, "%s", str);
```

Comment:
I think str variable is not needed.
you can put VFS_TEST_CONTENTS_1 here or change strlen(str) below.